products/android_studio.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Android Studio Plugin Development
+---
+
+## Introduction 
+Android Studio plugins can be created using IntelliJ IDEA, Java or Kotlin, and the IntelliJ Platform.
+They can also be packaged and distributed at [plugins.jetbrains.com](https://plugins.jetbrains.com).
+
+Android Studio plugins extend or add functionality to the Android Studio IDE.
+They are not Android modules or apps to run in the Android operating system, such as on a smartphone or tablet.
+
+## Configuring IntelliJ Platform Projects for Android Studio Plugin Development
+To create a new Android Studio plugin project, follow the tutorial on the [Getting Started with Gradle](/tutorials/build_system/prerequisites.md) page.
+The tutorial produces a skeleton project suitable to use as a starting point for an Android Studio plugin.
+On the [New Project Configuration Screen](/tutorials/build_system/prerequisites.md#new-project-configuration-screen) of the New Project Wizard tutorial, choose Gradle from the product category pane as described in the tutorial, **not** _Android_.
+  
+Some minor modifications to the skeleton project are needed, as discussed below.
+
+### Matching Versions of the IntelliJ Platform with the Android Studio Version
+For API compatibility, it is essential to match the version of the IntelliJ Platform APIs used for plugin development with the target version of Android Studio.
+The version number of Android Studio contains the version of the underlying IntelliJ Platform APIs that were used to build it.
+
+To find the version of the IntelliJ Platform used to build Android Studio, use the Android _About_ dialog screen.
+An example is shown below.
+In this case, the (BRANCH.BUILD.FIX) version of the IntelliJ Platform is `191.8026.42`, which corresponds to the IntelliJ IDEA version 2019.1.4.
+The [`build.gradle` configuration steps](#configuring-the-plugin-buildgradle-file) below explain how to set the IntelliJ Platform version to match the target version of Android Studio.
+
+![Example Android Studio About Dialog](img/android_studio_build.png){:width="600px"}
+
+Using Gradle as the basis for developing Android Studio plugins means the version of the IntelliJ IDEA IDE used to do the actual development does not have to match the targeted version of Android Studio.
+They are independent because the version of the IntelliJ Platform API used when developing a plugin can be [set independently](/tutorials/build_system/gradle_guide.html#intellij-platform-configuration) of the version of the IntelliJ IDEA IDE being used to do the development.
```

Comment:
I don't think it's very clear what is the role Gradle covers in the independence from the IDE version. Maybe expand a bit on that? You may also reword these two lines a bit because just reading the first one left me going back to try and parse it.

Reply:
You are right. I prefer not to address that issue here, but rather in the destination of the link so it is shared across the other products as well. Before I do that I'd like to talk with Yann and Alexander.

Consequently, I'm removing the section for now.